backend/src/services/mail-verification/send-activation-link/templates/email.hbs

```diff
@@ -31,11 +31,11 @@
                     <path d="M85.6931 17.3739V20.4354H89.8171V22.6865H85.6931V25.811H91.0777V28.1522H82.6315V15.0327H91.0777V17.3739H85.6931Z" fill="#3765FF"/>
                     </svg>
           </p>
-          <p style="text-align:left">Hello.<br><br>Follow the link below to verificate your email.
+          <p style="text-align:left">Hello.<br><br>Thanks for signing up! Please follow the link below to verify your email address and access all website functionality.
```

Comment:
```suggestion
          <p style="text-align:left">Hello!<br><br>Thanks for signing up! Please follow the link below to verify your email address and access all website functionality.
```

Could you please apply the @setar92 change and we should be good to go w/ this PR